pkg/util/validation/user_limits_handler.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/pkg/util/validation/user_limits_exposition.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+
```

Comment:
We only need to include these "Provenance" headers if the file was copied from Cortex. That's not the case here, so we can remove all Provenance headers here.

Same in `user_limits_handler_test.go`.